### ISSUE TITLE
Documentation: Homework Assignment User Story

### ISSUE THREAD

**Issue by andrewstec** [ubccpsctech/PrairieLearn#21]
As a faculty member, I would like to create a homework assignment in Prairie Learn, so that students can complete the assignment using Prairie Learn.

Ready for Development: 

- Has access to course Github repository on [Github.ubc.ca/PrairieLearn](https://github.ubc.ca/PrairieLearn/).
- Repository above is loadable into Prairie Learn _if_ pre-existing course code exists.
- Faculty member has access to Prairie Learn in LDAP.
- If homework assignment should contain any open-ended manually graded questions, then the instructor and TAs have access to an appropriate platform compatible with Prairie Learn's API to export questions to.

Requirements:

- An [Assessment](https://prairielearn.readthedocs.io/en/latest/assessment/), which will become the homework assignment, is created and configured according to the documentation.
- [Questions](https://prairielearn.readthedocs.io/en/latest/question/) for the assignment are written or sourced from course question pool. See [Question Writing](https://todo/) user story.
- Questions for the assignment are implemented in the assessment.
- [Assessment Access Controls](https://prairielearn.readthedocs.io/en/latest/accessControl/) are set to allow only course students to access assignment during appropriate period.

Nice to Have's: 

- Late submission rules, if any required, are set.

Definition of Done: 

- Course repository loads into PrairieLearn development environment successfully.
- Homework assignment can be viewed in Prairie Learn.
- Intended questions appear in homework assignment.
- Newly written elements and questions have been manually tested in Prairie Learn.
- Newly written elements and questions have been automatically tested in Prairie Learn underneath Question Settings area.
- Student access to the assessment has been set with access controls.
- Any open-ended written questions that require manual grading are exportable to Canvas so that they can be manually graded by an instructor or TA.

Ready for Development:

- Must know how to write a question in Prairie Learn
- Must understand how access controls work.

Blocked By: 

- Requesting Prairie Learn from Tech Staff.
- Setting up your development environment.
- Setting up a Prairie Learn course or Continuing a Prairie Learn course.
- Student, Instructor, and TA access to Prairie Learn.
